spring-boot-starter/example-autodeployment/README.md

```diff
@@ -36,7 +36,7 @@ public class AutoDeploymentApplication {
 
 3. You can also put BPMN, CMMN and DMN files in your classpath, they will be automatically deployed and registered within the process application.
 
-4. When implementing the process application in [standard manner](https://docs.camunda.org/manual/7.12/user-guide/process-applications/the-process-application-class/),
+4. When implementing the process application in [standard manner](https://docs.camunda.org/manual/latest/user-guide/process-applications/the-process-application-class/),
```

Comment:
The link should point to the 7.13 version of the docs since this version of the examples might be checked-out and used after several releases. In that case, using latest wouldn't be valid.

```suggestion
4. When implementing the process application in [standard manner](https://docs.camunda.org/manual/7.13/user-guide/process-applications/the-process-application-class/),
```